src/chainparams.cpp

```diff
@@ -150,6 +164,17 @@ class CTestNetParams : public CMainParams {
         nMinerThreads = 0;
         nPruneAfterHeight = 1000;
 
+        // 1MB max blocks before 1 Aug 2015
+        // Then, if miner consensus: 8MB max, doubling every two years
+        nMaxSizePreFork = 1000*1000; // 1MB max pre-fork
+        nEarliestSizeForkTime = 1438387200; // 1 Aug 2015 00:00:00 UTC
```

Comment:
There should be a nLatestSizeForkTime which is the latest the fork can be activated, and if not, the fork would fail.